photutils/aperture/tests/test_aperture_photometry.py

```diff
@@ -104,42 +104,6 @@ def test_aperture_pixel_positions():
 
 class BaseTestAperturePhotometry:
 
-    def test_scalar_error(self):
-        # Scalar error
-        error = 1.
```

Comment:
so this is not a valid use case any more, or other tests covering it, too?

Reply:
It was never documented, but apparently the code supported the use of scalar `error` inputs.  In the refactor I removed the support of scalar `error` (it doesn't seem very useful in practice and removing this feature makes it consistent with other `photutils` functions).  I didn't mention this change in the change log because it was never advertised as a feature in the docs.

Reply:
ok, sounds good then. I lost track of the  features supposed to be used and the ones that are just tag along long time ago.

Reply:
Me too. :)